src/Components/CreateImageWizard/formComponents/Oscap.js

```diff
@@ -87,8 +87,8 @@ const PolicySelector = ({ input, showSelector }) => {
         onClear={handleClear}
         selections={policy}
```

Comment:
Do we want to switch "policy" for "profile" in the names of variables as well? It will take more work, but if the change is here to stay it could be much less confusing in the long run, what do you think?

Reply:
Yes, in fact it makes no sense that this is called policy. I'm going to change all of that